src/meta/src/rpc/server.rs

```diff
@@ -483,7 +486,7 @@ pub async fn rpc_serve_with_store<S: MetaStore>(
             ClusterManager::start_heartbeat_checker(cluster_manager, Duration::from_secs(1)).await,
         );
 
-        sub_tasks.push(GlobalBarrierManager::start(barrier_manager).await);
+        sub_tasks.push(GlobalBarrierManager::start(barrier_manager, split_diff_rx).await);
```

Comment:
@yezizp2012  a little off the topic, why GlobalBarrierManager does not start in test mode (`#[cfg(not(test))]`).

Reply:
Well, `rpc_serve_with_store` is only used in `server.rs` in the test mode and few things are tested. Actually all sub tasks can be configured to not start if we don't care about it and need it in the test mode.